bin/local-env/docker-compose.yml

```diff
@@ -27,9 +28,11 @@ services:
     volumes:
       - wordpress_data:/var/www/html
       - ../../:/var/www/html/wp-content/plugins/google-site-kit
+      - ../../tests/e2e/mu-plugins:/var/www/html/wp-content/mu-plugins
     depends_on:
       - mysql
       - wordpress
+    command: tail -f /dev/null
```

Comment:
What's the thinking behind this?

Reply:
It needs a command running in the foreground that will keep the container from stopping so that we can use `exec` to run commands inside it.

Reply:
Seems hacky, but makes sense :-)